.github/workflows/update-base-image.yml

```diff
@@ -0,0 +1,55 @@
+name: "Update image and push to Github Packages and Docker Hub weekly"
+
+on:
+  schedule:
+    - cron: "0 12 * * 1" # Run every Monday at noon.
```

Comment:
Forgive me for my lack of knowledge but why is there a need to rebuild the image weekly if nothing changes?

Is it to incorporate changes from the base image? Is this a good practice?

Reply:
Thanks for the explanation. I'll trust your judgment on this!